### PR TITLE
feat(expansion-panel): show tooltip for title and description

### LINKED ISSUE #12763: Tooltip for the full subtitle/title on hover for the Expansion panel
## Is your feature request related to a problem? Please describe.
When Expansion Panel has long title or subtitle, in the preview it should be shown on hover. 

## Describe the solution you'd like
On hover I want to see a preview (tooltip) of the full text

## Additional context
[Stackblitz sample to observe the missing functionality ](https://stackblitz.com/edit/angular-afgbkw?file=src%2Fapp%2Fexpansion-sample-2%2Fexpansion-sample-2.component.html,src%2Fapp%2Fexpansion-sample-2%2Fexpansion-sample-2.component.ts)

### PR DESCRIPTION
**Pull request by RivaIvanova**
Closes #12763   

### Additional information (check all that apply):
 - [ ] Bug fix
 - [x] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [x] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [x] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Review by jackofdiamond5 — APPROVED** [review]

**Review by jackofdiamond5 — DISMISSED** [review]

File: projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.directives.ts
Comment (kdinev) [inline projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.directives.ts:24]: Create a base class with this method and extend both directives from this new class.

**Review by kdinev — CHANGES_REQUESTED** [review]

**Comment by RivaIvanova** [comment]
@kdinev It is ready, however, the `do not merge` label is due to the mass merge to master https://github.com/IgniteUI/igniteui-angular/pull/12935, and as far as I know we should wait for it before merging PRs to master.

**Pull request merged**
